src/components/shopFormUIElements.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import star from "../../public/images/star.png";
+import signUpFormBar from "../../public/images/signUpFormBar.png";
+import Image from "next/image";
+
+export default function ShopFormUIElements() {
+  return (
+    <>
+      <div className="absolute left-32 top-16 h-16 w-16">
+        <Image src={star} />
+      </div>
+
+      <div className="absolute right-32 bottom-40 h-16 w-16">
+        <Image src={star} />
+      </div>
+      <Image src={signUpFormBar} />
```

Comment:
I didn't have time to do progress bar anymore and focused on learning new tech AHAHAH but I think it could be done with a css grid with color change if I had more time!